### PR TITLE
Small API test improvement for compatibility search endpoint

### PR DESCRIPTION
**Pull request by mlegenovic**
Signed-off-by: Milivoje Legenovic <m.legenovic@gmail.com>

### CODE REVIEW

File: test/apiv2/rest_api/test_rest_v2_0_0.py [hunk 1]
```diff
@@ -401,19 +401,6 @@ def do_search5():
...
-            r = requests.get(PODMAN_URL + "/v1.40/images/search?term=alpine", timeout=5)
-            self.assertEqual(r.status_code, 200, r.text)
-            objs = json.loads(r.text)
-            self.assertIn(type(objs), (list,))
-
-        search = Process(target=do_search)
-        search.start()
-        search.join(timeout=10)
-        self.assertFalse(search.is_alive(), "/images/search took too long")
-
```
Comment (mlegenovic) [inline test/apiv2/rest_api/test_rest_v2_0_0.py:416]: This block was included by accident in previous PR.

**Comment by rhatdan** [comment]
LGTM

**Pull request merged**